website/landingpage/dev/src/pages/about.html

```diff
@@ -0,0 +1,89 @@
+<section>
+  <div class="row">
+    <div class="column small-12 large-6">
+      <div class="sub-title has-number"><span class="text">Introducing Luigi</span></div>
+      <div class="title-2">
+        A better answer to distributed UI development - secure and infinitely scalable.
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="columns xlarge-9 content-wrapper" id="stickyAnchor" data-something>
+      <p>Offering a wide array of products, many companies are also offering just as many entry points as user interfaces. Oftentimes, these entry points and user interfaces lack in consistency and recognizability. Seamless transition between products is often hard to achieve, and this is one of the cornerstones of a great customer experience. Ensuring a consistent look and feel and harmonizing all the products and applications is a demanding and an expensive task. It means rebuilding them from the ground up and accommodating to particular UX requirements. One of the reasons why it is so difficult is that different products are developed on different technology stacks.</p>
```

Comment:
```suggestion
      <p>Offering a wide array of products, many companies are also offering just as many entry points as user interfaces. Oftentimes, these entry points and user interfaces lack in consistency and recognizability. Seamless transition between products is often hard to achieve, and this is one of the cornerstones of a great customer experience. Ensuring a consistent look and feel and harmonizing all the products and applications is a demanding and expensive task. It means rebuilding them from the ground up and fulfilling particular UX requirements. One of the reasons why it is so difficult is that different products are developed on different technology stacks.</p>
```

Reply:
I added some language comments but feel free to ignore them/consult with someone else first 🙂